app/javascript/app/layouts/embed/embed-component.jsx

```diff
@@ -2,16 +2,25 @@ import React, { PureComponent } from 'react';
 import Proptypes from 'prop-types';
 import { renderRoutes } from 'react-router-config';
 import CountriesProvider from 'providers/countries-provider';
+import cwLogo from 'assets/icons/cw-logo.svg';
+import Icon from 'components/icon';
 
 import layout from 'styles/layout';
+import styles from './embed-styles.scss';
 
 class Embed extends PureComponent {
   render() {
     const { route } = this.props;
     return (
-      <div className={layout.content}>
+      <div className={[layout.content, styles.embed]}>
         <CountriesProvider />
-        {renderRoutes(route.routes)}
+        <div className={styles.embedContent}>{renderRoutes(route.routes)}</div>
+        <div className={styles.footer}>
+          For more information, visit
+          <a href="https://www.climatewatchdata.org/" parent="_blank">
```

Comment:
When clicking on the link it is not redirecting to ClimateWatch website.  
Instead of ``parent="_blank"`` shouldn't it be ``target="_blank"``?